src/runtime/cmd/kata-runtime/kata-check.go

```diff
@@ -87,6 +89,108 @@ var (
 	kvmDevice = "/dev/kvm"
 )
 
+// Generated from https://zhwt.github.io/yaml-to-go/
+type ThirdPartyVersions struct {
+	Externals struct {
```

Comment:
Should only containerd and CRI-O be kept and remove others?

Reply:
Makes sense